contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -241,6 +249,15 @@ contract VRFV2Plus is BaseTest {
       extraArgs: VRFV2PlusClient._argsToBytes(VRFV2PlusClient.ExtraArgsV1({nativePayment: true}))
     });
     (, uint96 ethBalanceBefore, , ) = s_testCoordinator.getSubscription(subId);
+    vm.expectEmit(true, true, true, true);
```

Comment:
this can be `(true, false, false ,true)` right? since there is no second and third indexed fields.
If we make subID field indexed, then this can be `(true, true, false ,true)`

Reply:
Yes, sure